boards/FSM/Src/Io/Io_Coolant.c

```diff
@@ -37,46 +38,50 @@ void Io_FlowMeter_CheckIfFlowMeterIsActive(void)
     Io_SharedFreqOnlyPwmInput_CheckIfPwmIsActive(flow_meter);
 }
 
+float voltageToTemperature(float v)
+{
+    // TODO calculate the temperature
+    return v;
+}
 float Io_Coolant_TemperatureAVoltage(void)
 {
-    return 0.5f;
+    return Io_Adc_GetChannelVoltage(ADC_CHANNEL_4);
 }
 float Io_Coolant_GetTemperatureA(void)
 {
     float temperature_voltage = Io_Coolant_TemperatureAVoltage();
-    // TODO calculate the temperature
-    return 1;
+    return voltageToTemperature(temperature_voltage);
 }
-
 float Io_Coolant_TemperatureBVoltage(void)
 {
-    return 0.5f;
+    return Io_Adc_GetChannelVoltage(ADC_CHANNEL_5);
 }
 float Io_Coolant_GetTemperatureB(void)
 {
     float temperature_voltage = Io_Coolant_TemperatureBVoltage();
-    // TODO calculate the temperature
-    return 1;
+    return voltageToTemperature(temperature_voltage);
 }
 
+float voltageToPressure(float v)
+{
+    // TODO calculate the pressure
+    return v;
+}
```

Comment:
What's the plan for these? Another PR
(same with temperature above)

Reply:
Joe said we don't know what sensors we are using yet. Once that gets decided, I will implement the transfer functions